<!-- -*- mode: XML -*- -->
<launch>
  <!--load config file to configurate the dmx_firmware /-->
  <rosparam file="$(find dynamixel_motors_driver)/config/dmx_firmware_config.yaml" command="load"/>
  <!--launch dmx position controller node/-->
  <node name="position_control" pkg="dynamixel_motors_driver" type="position_control" required="true" output="screen"/>
  <!--launch firmware manager/-->
  <node name="dmx_manager" pkg="dynamixel_motors_driver" type="dmx_manager.py" output="screen"/>
  
</launch>
